packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -84,142 +89,65 @@ fn get_pkg_path(name: &str, pkg_type: PackageType) -> String {
 
 /// TODO: DOC add docstrings on impl'd methods
 impl<'s> JsPackage<'s> {
-    fn import(
+    fn import_package(
         scope: &mut v8::HandleScope<'s>,
-        context: v8::Local<'s, v8::Context>,
-        embedded: &Embedded<'s>,
         name: &str,
         pkg_type: PackageType,
     ) -> Result<Self> {
         let path = get_pkg_path(name, pkg_type);
         tracing::debug!("Importing package from path `{path}`");
-        let code = match fs::read_to_string(path.clone()) {
+
+        let namespace: Object<'_> = match import_and_get_module_namespace(scope, &path) {
             Ok(s) => s,
-            Err(_) => {
+            Err(Error::AccessJavascriptImport(..)) => {
```

Comment:
Applied in https://github.com/hashintel/hash/pull/493/commits/a8ecd933b513e37a27ee143b4f29ea27dd09432e

Reply:
Let's not print the file path, though, as it's already printed right before and can get pretty long.

```
     0.064053098s DEBUG hash_engine_lib::worker::runner::javascript: Importing package from path `./src/simulation/package/init/packages/json/package.js`
    at src/worker/runner/javascript/mod.rs:98

     0.064115407s DEBUG hash_engine_lib::worker::runner::javascript: Couldn't read package file. It might intentionally not exist.
    at src/worker/runner/javascript/mod.rs:103

```

Reply:
Removed the path https://github.com/hashintel/hash/pull/493/commits/784fafc6e43b61f6be3b29d973aab280b8b8be3f